README.md

```diff
@@ -47,6 +47,7 @@ Command | Description | Arguments | Admin | Example
 --------|-------------|-----------|-------|--------
 **add** | Adds audio from a url to the song queue. If no songs are currently in the queue, the audio will begin playing immediately. Playlists may also be added using this command. The maximum amount of songs that can be added from a playlist is specified in `mumbledj.gcfg`. | youtube_video_url OR youtube_playlist_url OR soundcloud_track_url OR soundcloud_playlist_url | No | `!add https://www.youtube.com/watch?v=5xfEr2Oxdys`
 **addnext** | Adds audio from a url to the song queue after the current song. If no songs are currently in the queue, the audio will begin playing immediately. Playlists may also be added using this command. The maximum amount of songs that can be added from a playlist is specified in `mumbledj.gcfg`. | youtube_video_url OR youtube_playlist_url OR soundcloud_track_url OR soundcloud_playlist_url | Yes | `!addnext https://www.youtube.com/watch?v=5xfEr2Oxdys`
+**search** | Searches for a query in the specific service and add the first Video/Music found to the queue, as long as it's playable according to the MaxSongDuration set in `mumbledj.gcfg`. | yt OR sc AND query | No | `!search yt nyan cat`
```

Comment:
Two things:
* Small grammar change for the description: "Searches for a query in the specific service and adds the first item found to the queue as long as it is playable according to the MaxSongDuration set in `mumbledj.gcfg`.
* I feel like simply spelling out "youtube" and "soundcloud" is a bit clearer than abbreviating them to "yt" and "sc". Although it is slightly more typing I'm willing to bet that most people would first attempt to use the un-abbreviated forms.